ComposeAdvanced/app/src/main/java/com/example/android/wearable/composeadvanced/presentation/components/CustomTimeText.kt

```diff
@@ -61,6 +75,24 @@ fun CustomTimeText(
                     )
                 }
             } else null,
+            trailingCurvedContent = if (showWarning) {
+                {
+                    curvedText(
+                        text = debugWarning!!,
+                        style = CurvedTextStyle(textStyle),
+                        color = Color.Red
+                    )
+                }
+            } else null,
+            trailingLinearContent = if (showWarning) {
+                {
+                    Text(
+                        text = debugWarning!!,
```

Comment:
This is to workaround a compiler bug

<img width="777" alt="image" src="https://user-images.githubusercontent.com/231923/161606308-42a1b31c-6459-47ec-9b26-cf92a0428aec.png">

Reply:
https://youtrack.jetbrains.com/issue/KT-51838